deploy/kubernetes/Jenkinsfile.yml

```diff
@@ -18,5 +18,5 @@ stages:
             make check-no-changes;
         stage: lint
     k8sTest:
-        k8sTest: "v1.18.2,v1.17.2,v1.16.4,v1.15.7,v1.14.10"
+        k8sTest: "v1.20.7,v1.19.11,v1.18.2,v1.17.2,v1.16.4,v1.15.7,v1.14.10"
```

Comment:
Maybe we can start dropping some of the old versions from the matrix? I would leave the older one we support, but we can consider removing some of the others.

Reply:
I removed 1.15 and 1.16